src/api/controllers/articleController.js

```diff
@@ -0,0 +1,63 @@
+import Sequelize from 'sequelize';
+
+import models from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errors from '../../helpers/constants/error.messages';
+
+const userAttrib = ['username', 'profileImage', 'email'];
```

Comment:
@alainburindi why not use `userAttribute` instead of `userAttrib`? same for `articleAttrib`